Source/Csla.test/ValidationRules/ValidationTests.cs

```diff
@@ -768,14 +768,11 @@ public class HasLazyField : BusinessBase<HasLazyField>
     public static PropertyInfo<string> Value1Property = RegisterProperty<string>(nameof(Value1), RelationshipTypes.LazyLoad);
     public string Value1
     {
-      get
-      {
-        return LazyGetProperty(Value1Property, () => string.Empty);
-        //if (!FieldManager.FieldExists(Value1Property))
-        //  SetProperty(Value1Property, string.Empty);
-        //return GetProperty(Value1Property);
-      }
-      set { SetProperty(Value1Property, value); }
+      get => LazyGetProperty(Value1Property, () => string.Empty);
+      //if (!FieldManager.FieldExists(Value1Property))
```

Comment:
This is now confusing with the change. This belongs to the `get` but reads without knowledge as it belongs to the `set`. So just delete it.